### ISSUE TITLE
`computeAccessibleName` won't compute button name from child title

### ISSUE THREAD

**Issue by Cipscis** [eps1lon/dom-accessibility-api#1016]
Elements with the "button" role support [determining their accessible name from their content](https://www.w3.org/TR/wai-aria/#namefromcontent), and it's possible for internal elements to have no text content but instead specify their accessible name via a "title" attribute. For example:

```html
<button>
    <span title="Name from child title"></span>
</button>
```

This can come up, for example, when using internal elements that display an icon.

Unfortunately, calling `computeAccessibleName` on such a button returns an empty string, whereas it should return `'Name from child title'`. The correct title can be viewed, for example, in the accessibility tree of Chrome devtools:

![image](https://github.com/eps1lon/dom-accessibility-api/assets/117460/09d49d72-6426-471a-934a-160dfca19d2e)
![image](https://github.com/eps1lon/dom-accessibility-api/assets/117460/b386b863-1579-4070-ac8f-1625b51407ef)

Calling `computeAccessibleName` on the inner element returns the correct accessible name.

I've set up a replication in Codepen: [dom-accessibility-api accessible name bug reproduction](https://codepen.io/Cipscis/pen/vYbajaE)

**Comment by neaumusic** [comment]
Yep having this same issue -- check out a hackernews 'upvote' button/link, you'll notice the title is not included in the computed accessible name nor description
